src/lib/components/List/List.svelte

```diff
@@ -0,0 +1,39 @@
+<script lang="ts">
+	import { type Variant, listVariants } from '../../../styles/tailwind/list';
+	import { typography } from '../../../styles/tailwind/typography';
+
+	export let title: string;
+	export let createdAt: string;
+	export let createdBy: string;
+	export let linkHref: string;
+	export let variant: Variant;
+
+	function formatDate(dateString: string): string {
+		const date = new Date(dateString);
+		const options: Intl.DateTimeFormatOptions = {
+			day: '2-digit',
+			month: 'long',
+			year: 'numeric',
+			calendar: 'buddhist'
+		};
+		return new Intl.DateTimeFormat('th-TH', options).format(date);
+	}
+</script>
+
+<a
+	href={linkHref}
+	target="_blank"
+	class={`h-auto w-[761px] flex flex-col py-[12px] px-[24px] max-md:w-[390px] rounded ${listVariants[variant]}`}
```

Comment:
trying to use [tailwind-unit](https://tailwindcss.com/docs/padding) as possible na for code clarity 

`py-[12px]`  -> `py-3`
`px-[24px]` -> `px-6`

คิดง่ายๆว่า แบบ 12 / 4  = 3
24 / 4 = 6

แต่ว่าแบบอะไรที่มันแปลงเป็น tailwind unit ไม่ได้ก็นั่นละเป็น `px` ไปแม่ง